index.html

```diff
@@ -3,7 +3,7 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
 
-<title>Thinkful React Starter</title>
+<title>GitHub Search App</title>
```

Comment:
```suggestion
<title>GitHub Search App</title>
```
<title>GitHub Search Appdf</title>